migration/v1_to_v2/data/exporters/data_exporter.rb

```diff
@@ -143,14 +143,8 @@ def value_to_ruby_string(value, include_blank = false)
   # rubocop:enable Metrics/MethodLength
   # rubocop:enable Metrics/AbcSize
 
-  def valid_key?(key)
-    return false if key.is_a?(Integer) || key.include?('-') || key.include?(' ') || !key.match?('^[a-zA-Z]')
-
-    true
-  end
-
   def key_to_ruby(key)
-    valid_key?(key) ? key : "'#{key}'"
+    "'#{ key.gsub(/'/, "\'") }'"
```

Comment:
"'#{ key.gsub(/'/, "\'") }'"
I think need to add one more \
like:
"'#{ key.gsub(/'/, "\\'") }'"